.github/workflows/post-pr-comment.yml

```diff
@@ -0,0 +1,30 @@
+name: Post pull-request comment
```

Comment:
Minor: Mention query help somewhere.
```suggestion
name: Query help preview - post pull-request comment
```

Reply:
This workflow is generic and can be used by more workflows in the future.

Reply:
Good point!